docs/apache-airflow/issue-triage-process.rst

```diff
@@ -0,0 +1,195 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+
+Issue reporting and resolution process
+======================================
+
+This document explains the issue tracking and triage process within Apache
+Airflow including labels, milestones, and priorities as well as the process
+of resolving issues.
+
+Labels
+''''''
+
+Since Apache Airflow uses Github Issues as the issue tracking system, the
+use of labels is extensive. Though issue labels tend to change over time
+based on components within the project, the majority of the ones listed
+below should stand the test of time.
+
+The intention with the use of labels with the Apache Airflow project is
+that they should ideally be non-temporal in nature and primarily used
+to indicate the following elements:
+
+**Kind**
+
+The “kind” labels indicate “what kind of issue it is”. The most
+commonly used “kind” labels are: bug, feature, documentation, or task.
+
+Therefore, when reporting an issue, the label of ``kind:bug`` is to
+indicate a problem with the functionality, whereas the label of
+``kind:feature`` is a desire to extend the functionality.
+
+There has been discussion within the project about whether to separate
+the desire for “new features” from “enhancements to existing features”,
+but in practice most “feature requests” are actually enhancement requests,
+so we decided to combine them both into ``kind:feature``.
+
+We recently introduced ``kind:task`` to categorize issues which are
```

Comment:
```suggestion
The ``kind:task`` is used to categorize issues which are
```
i'm not sure if it matters that this was recently added? Also, saves the future issue of fixing the sentence when "recently" is not true any longer.

Reply:
Done.